slack_lambda/main.tf

```diff
@@ -64,14 +64,18 @@ locals {
     #!/usr/bin/python3.6
     import urllib3
     import json
+    import os
+    slackChannel = os.environ['slack_channel']
```

Comment:
This makes sense but can we have it fallback to using TF replacement?
~~~
slackChannel = os.environ.get('slack_channel', "#${var.slack_channel}")
slackUsername = os.environ.get('slack_username', "${var.slack_username}")
slackIcon = os.environ.get('slack_icon', "${var.slack_icon}")
~~~

Reply:
Sometimes I thick skull.  Why, it doesn't work the way it was.

Reply:
Read the whole PR, Paul.   Nevermind.